content/en/observability_pipelines/guide/set_quotas_for_data_sent_to_a_destination.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Set Quotas for Data Sent to a Destination
+kind: documentation
+disable_toc: false
+private: true
+is_beta: true
+further_reading:
+- link: "/observability_pipelines/setup/"
+  tag: "Documentation"
+  text: "Set up Observability Pipelines"
+- link: "/observability_pipelines/working_with_data/"
+  tag: "Documentation"
+  text: "Working with data in Observability Pipelines"
+- link: "/monitors/configuration/"
+  tag: "Documentation"
+  text: "Learn more about configuring monitors"
+---
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSfnNnV823zAgOCowCYuXJE5cDtRqIipKsYcNpaOo1LKpGfppA/viewform" btn_hidden="false" header="Request Access!">}}
+The <code>quota</code> transform is in private beta.
+{{< /callout >}}
+
+## Overview
+
+The Observability Pipelines `quota` transform allows you to set a limit on the volume of data sent to a destination within a specific timeframe. This can safeguard you from unexpected data surges that might impact your operating costs. With the transform, you can set up different ways of handling the data when the quota has been reached. For example, you can:
+
+- Reroute the data sent after the quota limit to another destination, such as a cold storage.
+- Sample the data sent after the quota limit so that less data is sent to the destination.
+- Drop the data sent after the quota limit.
+- Set up a monitor to alert you that the quota has been reached.
```

Comment:
let's move this to the top of the bullet list, and say "Set a 'soft' limit by setting up a monitor to alert you that the quota has been reached."